traveler_ui_kit/build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
     implementation "com.android.support:recyclerview-v7:${android_support_version}"
     implementation "com.android.support:design:${android_support_version}"
     implementation "com.android.support:recyclerview-v7:${android_support_version}"
+    implementation 'com.google.android.gms:play-services-maps:17.0.0'
+    implementation 'com.google.android.gms:play-services-location:17.0.0'
```

Comment:
There needs to be research on this before we settle on a version.

Which version are clients using mostly? Are they backwards compatible? What effort is required if they are currently using a different (older) version? Do we have to use these?